app/knight.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+
+class Knight:
+    def __init__(self, knight: dict) -> None:
+        self.knight_property = knight

```

Comment:
Code Style Issue: The variable name `knight_property` is not very descriptive. Consider changing it to something more descriptive, such as `knight_attributes` or `knight_stats`.